src/visualizations/store/adapters/dhis_highcharts/pie.js

```diff
@@ -1,8 +1,12 @@
 export default function(acc, seriesIds, categoryIds, idValueMap, metaData) {
-    acc.push(
-        ...seriesIds.map(seriesId => ({
-            name: metaData.items[seriesId].name,
-            y: parseFloat(idValueMap.get(seriesId)),
-        }))
-    )
-}
+    seriesIds.forEach(seriesId => {
+        const value = idValueMap.get(seriesId)
+
+        if (value) {
+            acc.push({
+                name: metaData.items[seriesId].name,
+                y: parseFloat(idValueMap.get(seriesId)),
```

Comment:
Shouldn't this be `y: parseFloat(value),` ?

Reply:
Or maybe even move the `parseFloat` to `const value` to make sure it parsed okay in the `if (value)` check?

Reply:
But then there is a bigger problem somewhere higher up and I don't want to camouflage that.